documentation/features/cpuset.md

```diff
@@ -0,0 +1,55 @@
+# Cpuset cgroup.
+
+From Kernel documentation:
+
+_" Cpusets provide a mechanism for assigning a set of CPUs and Memory Nodes to
+a set of tasks."_
+
+The Kata agent brings compatibility the cgroup cpuset CPU on the guest side.
+
+The cpuset CPU cgroup will be applied on two events:
+
+- containers creation.
+
+- container update.
+
+When the runtime request to apply cpuset cgroup to the agent, the amout of
+vcpus available may be not the same to the required in the request.
+
+This is becasue the request from the agent client (the Kata runtime) pass
```

Comment:
Lines 19-23 suggested rewrite: 
This is because the request from the agent client (i.e. the Kata runtime) passes cpusets that are requested to be placed on the host. This isolates the container workload on some specific host CPUs. The runtime passes the requested cpuset to the agent, which tries to apply the cgroup cpuset on the guest.